packages/didcomm/src/handlers/MessageHandler.ts

```diff
@@ -4,7 +4,7 @@ import type { InboundMessageContext, OutboundMessageContext } from '../models'
 export interface MessageHandler {
   readonly supportedMessages: readonly ConstructableAgentMessage[]
 
-  handle(messageContext: InboundMessageContext): Promise<OutboundMessageContext | void>
+  handle(messageContext: InboundMessageContext): Promise<OutboundMessageContext | undefined>
```

Comment:
This is a breaking change. I think a good one, but maybe you can add a changeset entry that messagehandler should return undefined if it doesn't want to response with a message

Reply:
I am currently changing all the handlers to support this. Which of the following approach is the the best/correct to go with ?
1. Make the function return type as `Promise<undefined>` 
```typescript
public async processPerform(): Promise<undefined> {

    ...
}  
```
2. Purposefully `return undefined` from the function
```typescript
public async processPerform() {

    ...
    return undefined
}
```

Reply:
I think option 2 makes sense!

Reply:
Added tis changes now